news/2024/2024-05-29-osu-catch-world-cup-2024-round-of-32-recap.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "osu!catch World Cup 2024: Round of 32 Recap"
+date: 2024-05-29 12:00:00 +0000
+---
+
+Welcome to the Round of 32 recap of the osu!catch 2024 World Cup! 
+
+![](/wiki/Tournaments/CWC/2024/img/cwc2024-banner.jpg)
+
+As the qualifiers concluded, the top 32 teams that would battle it out for the title of osu!catch world cup winner were announced. Top of the pack is the big favorite to win it all this year, the United States, qualifying 1st for the second year in a row with 7 top scores out of 9 maps. Joining them on the qualification podium this year are Poland and Finland, both teams getting their best qualification result to date.
+
+Going further down the list, we see Estonia qualifying 32nd just right above the cut off for the first time in ten years. Iceland, the sole debutant of this year, defied all expectations and ended up 30th on the qualification list and are making their world cup debut in the round of 32.
+
+Let's get into the results and the highlights of last week's matches.
+
+## Round of 32 results
+
+In case you missed any of the matches, or are just curious about general results, you can view all of it here.
+
+|                                 Team A |       |       | Team B                      |
```

Comment:
tables shouldn't be "beautified" https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#tables